src/services/Frontline.js

```diff
@@ -124,6 +133,56 @@ class Frontline {
       }
     }
   }
+
+  pagination() {
+    document.addEventListener("DOMContentLoaded", () => {
+      var page = 1;
+      var win = $(window);
+      var loadingNow = false;
+      var categoryId = $(loadInfinityClasses.category).data("id");
```

Comment:
There is no jquery in wundery-frontline so we shouldn't depend on it (this currently only works because the theme accidentally also includes jquery but this might change and therefore break at any moment)